internal/common/retrystrategy/retry_state.go

```diff
@@ -1,18 +1,20 @@
 package retrystrategy
 
 const (
-	RetryStrategyPendingState    = "PENDING"
-	RetryStrategyCompletedState  = "COMPLETED"
-	RetryStrategyErrorState      = "ERROR"
-	RetryStrategyPausedState     = "PAUSED"
-	RetryStrategyUpdatingState   = "UPDATING"
-	RetryStrategyDeletingState   = "DELETING"
-	RetryStrategyInitiatingState = "INITIATING"
-	RetryStrategyIdleState       = "IDLE"
-	RetryStrategyFailedState     = "FAILED"
-	RetryStrategyActiveState     = "ACTIVE"
-	RetryStrategyDeletedState    = "DELETED"
-
+	RetryStrategyActiveState            = "ACTIVE"
```

Comment:
nit: we might want to use enums, e.g.: https://www.sohamkamani.com/golang/enums/

Reply:
Keeping as is to avoid more changes